src/utils/conditionalRendering.ts

```diff
@@ -51,8 +50,7 @@ export function runConditionalRenderingRules(
         });
 
         if (connection.repeatingGroup.childGroupId) {
-          const childGroup: IRepeatingGroup | undefined =
-            repeatingGroups && repeatingGroups[`${connection.repeatingGroup.childGroupId}-${index}`];
+          const childGroup = repeatingGroups && repeatingGroups[`${connection.repeatingGroup.childGroupId}-${index}`];
           if (childGroup) {
             for (let childIndex = 0; childIndex <= childGroup?.index; childIndex++) {
               const connectionNestedCopy: IConditionalRenderingRule = JSON.parse(JSON.stringify(connectionCopy));
```

Comment:
Is there a reason not to switch to `structuredClone` here as well?

Reply:
Nah, don't think I touched that code - but fixing that now!